Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py

```diff
@@ -1149,15 +1146,18 @@ def fetch_incidents(client: Client,
 def main():
     os.environ['PAN_CREDENTIALS_DBFILE'] = os.path.join(gettempdir(), 'pancloud_credentials.json')
     params = demisto.params()
-    registration_id_and_url = params.get(REGISTRATION_ID_CONST).split('@')
+    registration_id_and_url = params.get('credentials_reg_id', {}).get('password') or params.get('reg_id')
```

Comment:
You don't need the split?

Reply:
After doing some tests on this I saw that if it is still none it would fail on the split but not on the error ''Key, Token and ID must be provided'.
Therefore I added the split after the check that is not null - the split action is at line 1155